home.html

```diff
@@ -0,0 +1,164 @@
+                About
+            </li></a>
+           <a href="#program"> <li class="navigationItem">
+                Program
+            </li></a>
+            <a href="#join"> <li class="navigationItem">
+                Join
+            </li></a>
+           <a href="#sponsor">
+           <a href="#sponsor"> <li class="navigationItem">
+                Sponsor
+            </li>
+        </a>
+        </a>
+           <a href="#news"> <li class="navigationItem">
+                News
+            </li>
+        </a>
+        </ul>
+    </div>
+<header class="header flex">
+    <h1 class="mainTitle">
+        "Mellon!"
+    </h1>
+    <h2 class="secondTitle">
+        ARE YOU READY  A JOURNEY TO  MIDDLE-EARTH?
+    </h2>
+</header>
+<main>
+    <div class="index flex">
+        <p class="description">
+            As a Tolkien reader or  a Lord of the Rings watcher, we invite you a fantastic journey in Istanbul with different people all over the world.   
+        </p>
+        <p class="date">
+            2024.03.01 (WED) - 05 (FRI)</p>
+        <p class="location">
+@Taksim Square of İstanbul in Turkey
+        </p>
+    </div>
+    <div class="program flex" id="program">
+        <h2 class="mainProgram">
+            Main Program
+        </h2>
+        <hr class="mainHr"/>
+        <ul class="schedule flex">
+            <li class="scheduleItems flex">
+                <img src="lecture-logo.png" class="scheduleItemImage">
+                <h3 class="scheduleTitle"> 
+                    Opening
+                 </h3> 
+                 <p class="scheduleIndex">
+                    As speaker grandson of J.R.R Tolkien Simon Tolkien
+                 </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="exhibition-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Video Shows
+            </h3>
+            <p class="scheduleIndex">
+                The highlights of series
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="workshop-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle">
+                
+                Workshops
+            </h2>
+            <p class="scheduleIndex">
+                Role playing games, Creating Middle-Earth musics
+            </p>
+        </li>
+        <li class="scheduleItems flex">
+            <img src="network-logo.png" class="scheduleItemImage">
+            <h3 class="scheduleTitle"> 
+                Consers
+            </h3>
+            <p class="scheduleIndex">
+                Sting, Joan Ibanez, Mahmut Orhan, Michael Jackson-Luciano Pavarotti, Nil Karaibrahimgil, Amy Winehouse
+            </p>
+        </li>
+        </ul>
+        <button class="joinButton" id="join">
+            Join!
+        </button>
+<a href="#program"> <p class="wholeProgram">SEE THE WHOLE PROGRAM</p></a>
+    </div>
```

Comment:
- [ ] The dark background for this section is supposed to stretch the whole width of the page and not have a margin/ padding left and right as it currently is in your design.
- [ ] Kindly ensure that you are using the correct fonts and text sizes for your section so that it matches the provided design in Behance.
- [ ] In your "Main Program" desktop version, kindly ensure that the individual cards have an on hover that has a highlight effect on the card when hovered over.

**This is how your design looks like:**

![image](https://user-images.githubusercontent.com/86318284/212125157-45118705-05e2-48b8-bc6d-947e987c7874.png)


**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212125221-b5377184-451b-4579-ba3b-3623fe7da3b3.png)

- [ ] On **Mobile view** kindly reduce the size of the images as well as the texts. The current size wraps the section and the design does not look as the one provided in the Behance design.

**This is your mobile view:**

![image](https://user-images.githubusercontent.com/86318284/212127460-bb59c65f-cd2e-48d5-8e7f-b23e8cb429e3.png)


**This is the Behance home page mobile view**

![image](https://user-images.githubusercontent.com/86318284/212127594-5e66bb20-c623-42a0-b721-72dc5c763edb.png)